client/src/pages/Camera.jsx

```diff
@@ -0,0 +1,149 @@
+          setLongitude(position.coords.longitude);
+        });
+      }
+    },
+    [webcamRef, setImage]
+  );
+
+  const handleRetake = () => {
+    setImage(null);
+    setLatitude("");
+    setLongitude("");
+    setTitle("");
+    setInformation("");
+    setArtist("");
+  };
+
+  useEffect(() => {
+    if (!image) {
+      setLatitude("");
+      setLongitude("");
+    }
+  }, [image]);
+
+  return (
+    <Form
+      method="POST"
+      encType="multipart/form-data"
+      action="/camera"
+      className="camera-modal"
+    >
+      <section className="webcam-container">
+        {image ? (
+          <>
+            <img src={image} alt="Captured" className="captured-image" />
+            <input type="hidden" name="pictureTaken" value={image} />
+            <input type="hidden" name="userId" value={auth.id} />
+            <input type="hidden" name="latitude" value={latitude} />
+            <input type="hidden" name="longitude" value={longitude} />
+          </>
+        ) : (
+          <Webcam
+            audio={false}
+            ref={webcamRef}
+            screenshotFormat="image/jpeg"
+            className="webcam"
+          />
+        )}
+      </section>
+
+      {image && (
+        <section className="info-section">
+          <p>Ajouter des informations complémentaires (facultatif)</p>
+          <label htmlFor="title">Titre</label>
+          <input
+            type="text"
+            name="title"
+            id="title"
+            value={title}
+            onChange={(e) => setTitle(e.target.value)}
+          />
+
+          <label htmlFor="artist">Artiste</label>
+          <input
+            type="text"
+            name="artist"
+            id="artist"
+            value={artist}
+            onChange={(e) => setArtist(e.target.value)}
+          />
+
+          <label htmlFor="information">Description</label>
+          <textarea
+            name="information"
+            id="information"
+            rows="3"
+            value={information}
+            onChange={(e) => setInformation(e.target.value)}
+          />
+        </section>
+      )}
+
+      {!image ? (
+        <section className="initial-buttons">
+          <button
+            type="button"
+            aria-labelledby="switch-button"
+            className="switch-button"
+          >
+            <MdCameraswitch className="switch-icon" />
+          </button>
+          <button
+            type="button"
+            aria-labelledby="camera-button"
+            className="camera-button"
+            onClick={capture}
+          />
+        </section>
+      ) : (
+        <section className="button-container">
+          <button
+            type="button"
+            aria-labelledby="retake-button"
+            className="button-retake-picture"
+            onClick={handleRetake}
+          >
+            Reprendre
+          </button>
+          <button
+            type="submit"
+            aria-labelledby="upload-button"
```

Comment:
aria-label, not aria-labelled-by
aria labels should be in French
aria labels should not mention the type of tag, so here aria label is not necessary